repo/packages/C/chronos/0/config.json

```diff
@@ -72,7 +72,7 @@
     "chronos/hostname": {
       "description": "The advertised hostname stored in ZooKeeper so another standby host can redirect to this elected leader (default = localhost)",
       "type": "string",
-      "default": "`curl http://169.254.169.254/latest/meta-data/public-hostname`"
+      "default": "$HOST"
```

Comment:
This looks pretty specific to aws. What's the priority for supporting other
clouds?

--sent from my phone
On Apr 24, 2015 7:03 PM, "Connor Doyle" <notifications@github.com> wrote:

> In repo/packages/C/chronos/0/config.json
> <https://github.com/mesosphere/universe/pull/91#discussion_r29092748>:
>
> > @@ -72,7 +72,7 @@
> >      "chronos/hostname": {
> >        "description": "The advertised hostname stored in ZooKeeper so another standby host can redirect to this elected leader (default = localhost)",
> >        "type": "string",
> > -      "default": "`curl http://169.254.169.254/latest/meta-data/public-hostname` <http://169.254.169.254/latest/meta-data/public-hostname>"
> > +      "default": "$HOST"
>
> Nice one!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/universe/pull/91/files#r29092748>.
>